DependencyInjection/KnpGaufretteExtension.php

```diff
@@ -23,8 +25,10 @@ class KnpGaufretteExtension extends Extension
     /**
      * Loads the extension
      *
-     * @param  array            $configs
-     * @param  ContainerBuilder $container
+     * @param array $configs
+     * @param ContainerBuilder $container
+     *
+     * @throws Exception
```

Comment:
Please remove changes on this docblock (it's useless/irrelevant). As well as the use for `Exception`.